fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -493,7 +493,12 @@ public void onCreate() {
                 active = false;
                 continue;
             }
-
+            if (table instanceof MaterializedView && !((MaterializedView) table).active) {
+                LOG.warn("tableName :{} is invalid. set materialized view:{} to invalid",
```

Comment:
if some base tables are mv, and the mvs is not active, I think it is better to make the creation of mv fail with notice message: "there are some base materialized views are not active" or something?

Reply:
If do that, will it cause checkpoint and fe restart failed?